brody_lab_to_nwb/msortedprocesseddatainterface.py

```diff
@@ -0,0 +1,143 @@
+"""Authors: Cody Baker."""
+import numpy as np
+
+from h5py import File
+from pynwb import NWBFile
+from nwb_conversion_tools.basedatainterface import BaseDataInterface
+
+
+class MSortedProcessedInterface(BaseDataInterface):
+    """Conversion class for processed behavioral data parsed from raw 'saved history'."""
+
+    @classmethod
+    def get_source_schema(cls):
+        source_schema = dict(
+            required=["file_path"],
+            properties=dict(
+                file_path=dict(
+                    type="string",
+                    format="file",
+                    description="Path to .mat file containing processed data."
+                )
+            ),
+            type="object",
+            additionalProperties=False
+        )
+        return source_schema
```

Comment:
Is this method necessary? Can we use the super?

Reply:
That would depend sensitively on the nwb-conversion-tools version. After PR #237 is merged on nwb-conversion-tools and the version updated here, it likely could. For now, on the version for this PR, I can guarantee this will run without error.